prediction/src/algorithms/segment/src/data_generation.py

```diff
@@ -0,0 +1,63 @@
+import glob
+import os
+
+import numpy as np
+import pylidc as pl
+
+
+def get_dicom_paths(in_docker=True):
+    """Return DICOM paths to all LIDC directories
+    e.g. ['../images_full/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/' \
+          '1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192']"""
+    if in_docker:
+        return glob.glob("../images_full/LIDC-IDRI-*/**/**")
```

Comment:
We should always be using ``os.path.join`` :)

Reply:
ping on this one? :)

Reply:
Done in 28660bf681dad4acae8e20ee15433d8a48884384